src/Permissions/Widgets/AppSettingsView.vala

```diff
@@ -30,7 +30,7 @@ public class Permissions.Widgets.AppSettingsView : Gtk.Grid {
 
         var homefolder_widget = new PermissionSettingsWidget (
             Plug.permission_names["filesystems=home"],
-            _("Access your entire home folder, including any hidden folders."),
+            _("Access your entire Home folder, including any hidden folders in it."),
```

Comment:
Not sure the last part is needed as it's implied, and sounds awkward to me when including it.

```suggestion
            _("Access your entire Home folder, including any hidden folders."),
```